src/SIL.XForge.Scripture/ClientApp/e2e/src/app.e2e-spec.ts

```diff
@@ -1,12 +1,14 @@
+import { browser, by, element, ExpectedConditions } from 'protractor';
 import { AppPage } from './app.po';
 import { LoginPage } from './login.po';
 
-describe('App', () => {
+describe('App', () => {  
   const constants = require('../testConstants.json');
-  const loginPage = new LoginPage();
+  const loginPage = new LoginPage();  
 
   it('should display welcome message containing the users name', async () => {
     await loginPage.loginAsAdmin();
+    await browser.wait(ExpectedConditions.visibilityOf(AppPage.homepage.loginLabel), constants.conditionTimeout);
```

Comment:
Move this to the end of `LoginPage.login` method. So that everyone who logs in won't have to remember to add this wait.

Reply:
Okay, we have moved this browser.wait in the "LoginPage.login" method. Thank you.